code/datums/traits/neutral.dm

```diff
@@ -410,3 +410,31 @@
 	desc = "You identify as a dog! (Mostly to help identify your species mechanically)"
 	value = 0
 	mob_trait = TRAIT_DOG
+
+/datum/quirk/item_quirk/photographer
+	name = "Photographer"
+	desc = "You carry your camera and personal photo album everywhere you go, and your scrapbooks are legendary among your coworkers."
+	value = 0
+	mob_trait = TRAIT_PHOTOGRAPHER
+	gain_text = span_notice("You know everything about photography.")
+	lose_text = span_danger("You forget how photo cameras work.")
+	medical_record_text = "Patient mentions photography as a stress-relieving hobby."
+
+/datum/quirk/item_quirk/photographer/add_unique(client/client_source)
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	var/obj/item/storage/photo_album/personal/photo_album = new(get_turf(human_holder))
```

Comment:
`var/obj/item/storage/photo_album/personal/photo_album`
to:
`var/obj/item/storage/photo_album/photo_album`